src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_abort.yaml

```diff
@@ -13,21 +13,21 @@ interactions:
       ParameterSetName:
       - --resource-group --name --ssh-key-value --no-wait
       User-Agent:
-      - AZURECLI/2.40.0 azsdk-python-azure-mgmt-resource/21.1.0b1 Python/3.8.10 (Linux-5.15.0-1019-azure-x86_64-with-glibc2.29)
+      - AZURECLI/2.39.0 azsdk-python-azure-mgmt-resource/21.1.0b1 Python/3.8.10 (Linux-5.13.0-1022-azure-x86_64-with-glibc2.29)
```

Comment:
Why these unrelated recordings got updated?

Reply:
OK, then reset all these recordings except the one `test_aks_create_or_update_with_load_balancer_backend_pool_type.yaml` you newly added?

Reply:
done